virtualenv/flavors/base.py

```diff
@@ -6,24 +6,95 @@
 
 class BaseFlavor(object):
 
-    core_modules = set([
-        "posixpath.py", "stat.py", "genericpath.py", "warnings.py",
-        "linecache.py", "types.py", "UserDict.py", "_abcoll.py", "abc.py",
-        "_weakrefset.py", "copy_reg.py",
-    ])
+    def bootstrap_modules(self, python_info):
```

Comment:
Only the available modules or directories (note: *not necessarily packages*) will be copied.